sdk/core/core-http/src/httpOperationResponse.ts

```diff
@@ -81,5 +81,8 @@ export interface RestResponse {
    */
   _response: HttpOperationResponse;
 
+  /**
+   * May contain other properties.
```

Comment:
Perhaps it may help to see this in context? What is valuable of allowing more properties here?